ihatemoney/templates/statistics.html

```diff
@@ -33,6 +33,243 @@ <h2>{{ _("Expenses by Month") }}</h2>
         {% endfor %}
         </tbody>
         </table>
+        <h2> {{ _("Visualization") }}</h2>
+        
+        
+        <!-- Required for some style in charts, feel free to move from here -->
+        <style>
+        #balanceBar .bb-ygrid-line line {stroke: red;}
+
+        #balanceBar .fill_green { fill: green; }
+        #balanceBar .fill_red { fill: red; }
+        </style>
+        <br>
+        <div id="balanceBar"></div>
+        <br>
+        <div id="memberPerMonth"></div>
+        <br>
+        <div id="combinedPerMonth"></div>
+        <br>
+        <div id="expPie"></div>
+        
+
+
+        
+        
+
+        
+
+        <script> 
```

Comment:
As this is quite a long `script` tag, I would prefer to isolate it in a separate file. It would also force us to avoid looping JavaScript statements in the template directly.